timelock-server/src/testCommon/java/com/palantir/atlasdb/timelock/TimeLockServerHolder.java

```diff
@@ -168,4 +159,13 @@ TimeLockInstallConfiguration installConfig() {
             throw new RuntimeException(e);
         }
     }
+
+    private static WireMockConfiguration wireMockConfiguration(int port) {
```

Comment:
Here's a fun one: I think this should take an `Integer` as opposed to an `int` (why?)

Reply:
because httpsport takes an `Integer`?